packages/block-editor/src/hooks/use-zoom-out.js

```diff
@@ -13,6 +14,16 @@ import { store as blockEditorStore } from '../store';
  * A hook used to set the editor mode to zoomed out mode, invoking the hook sets the mode.
  */
 export function useZoomOut() {
+	const isSiteEditor = getPath( window.location.href )?.includes(
+		'site-editor.php'
```

Comment:
I don't like this way of working out what editor context we are in. I think it should be possible to set the edit context in the blockEditor store, although I haven't yet found a way to do it!

Reply:
Agreed. I saw it used several times in the codebase already by @youknowriad. It is faster than adding a `useSelect` and does work in this context though...